hudi-flink/src/main/java/org/apache/hudi/source/StreamWriteFunction.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.operator;
+package org.apache.hudi.source;
```

Comment:
Should it be put into the `source` package?

Reply:
Nope, thanks for the reminder ~